anise/src/naif/kpl/tpc.rs

```diff
@@ -157,7 +157,7 @@ fn test_anise_conversion() {
     let path = "../target/gm_pck_08.anise";
 
     // Test saving
-    dataset.save_as(&PathBuf::from(path), true).unwrap();
+    dataset.save_as(&Path::from(path), true).unwrap();
```

Comment:
This doesn't seem to build because it's missing the Path import. But when I add the Path import, the compiler is still upset:

```
error[E0277]: the size for values of type `[u8]` cannot be known at compilation time
    --> anise/src/naif/kpl/tpc.rs:160:33
     |
160  |     dataset.save_as(&Path::from(path), true).unwrap();
     |                      ---------- ^^^^ doesn't have a size known at compile-time
     |                      |
     |                      required by a bound introduced by this call
```